### PR TITLE
Add tomb adventure start

### PR DESCRIPTION
**Pull request by timfernandez**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2554)

### CODE REVIEW

File: english/tomb/tombraider.md [hunk 1]
```diff
@@ -0,0 +1 @@
+after checking the playlist, the next mission popped up
```
Comment (casey-collab) [inline english/tomb/tombraider.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**